Integration tests involving PostgreSql and MySql images are failing

This is not an issue of the provisioning tooling since it works provided the PostgreSql and MySql images URL are correct.
Tests involving PostgreSql and MySql provisioning are failing at pulling the images from quay, e.g.:

```
...
[ERROR] Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 92.754 s <<< FAILURE! - in org.jboss.intersmash.testsuite.provision.openshift.KeycloakOperatorProvisionerTest
[ERROR] org.jboss.intersmash.testsuite.provision.openshift.KeycloakOperatorProvisionerTest.exampleSsoWithDatabase  Time elapsed: 2.594 s  <<< ERROR!
cz.xtf.core.waiting.WaiterException: 
Following events match condition: <at least one exists>
	2024-03-29T11:38:04Z	Pod/postgresql-1-2n7ml	Failed to pull image "quay.io/centos7/postgresql-13-centos7": rpc error: code = Unknown desc = reading manifest latest in quay.io/centos7/postgresql-13-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine
Filter:	 obj names: [postgresql.*]
	 messages: [Failed to pull image.*, Build.*failed, Error syncing pod, skipping: failed to.*, FailedMount MountVolume.SetUp.*, Error creating: pods.*is forbidden:.*, .*failed to fit in any node.*, Failed to attach volume.*]
	 after: 2024-03-29T11:38:02Z

	at org.jboss.intersmash.testsuite.provision.openshift.KeycloakOperatorProvisionerTest.exampleSsoWithDatabase(KeycloakOperatorProvisionerTest.java:224)
...
```

**To Reproduce**

- Execute the `PostgreSQLImageTestCase` test: `mvn test -pl testuite/integration-tests/ -amd -Dtest=PostgreSQLImageTestCase`, or:

- Try to pull the PostgreSql image:
```
$ podman pull quay.io/centos7/postgresql-13-centos7
Trying to pull quay.io/centos7/postgresql-13-centos7:latest...
WARN[0001] Failed, retrying in 1s ... (1/3). Error: initializing source docker://quay.io/centos7/postgresql-13-centos7:latest: reading manifest latest in quay.io/centos7/postgresql-13-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine 
WARN[0003] Failed, retrying in 1s ... (2/3). Error: initializing source docker://quay.io/centos7/postgresql-13-centos7:latest: reading manifest latest in quay.io/centos7/postgresql-13-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine 
```

- and similarly for the MySql image:
```
podman pull quay.io/centos7/mysql-80-centos7
Trying to pull quay.io/centos7/mysql-80-centos7:latest...
WARN[0001] Failed, retrying in 1s ... (1/3). Error: initializing source docker://quay.io/centos7/mysql-80-centos7:latest: reading manifest latest in quay.io/centos7/mysql-80-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine 
WARN[0004] Failed, retrying in 1s ... (2/3). Error: initializing source docker://quay.io/centos7/mysql-80-centos7:latest: reading manifest latest in quay.io/centos7/mysql-80-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine 
WARN[0006] Failed, retrying in 1s ... (3/3). Error: initializing source docker://quay.io/centos7/mysql-80-centos7:latest: reading manifest latest in quay.io/centos7/mysql-80-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine 
Error: initializing source docker://quay.io/centos7/mysql-80-centos7:latest: reading manifest latest in quay.io/centos7/mysql-80-centos7: unknown: Tag latest was deleted or has expired. To pull, revive via time machine
```

**Additional context**

This image is no longer supported, see https://quay.io/repository/centos7/postgresql-13-centos7, so a quick fix is to use the `:centos7`, while the actual solution would be to upgrade the supported PostgreSql image version, e.g.: https://quay.io/repository/sclorg/postgresql-15-c9s?tab=info

--- a/global-test.properties
+++ b/global-test.properties
@@ -13,8 +13,8 @@ xtf.junit.prebuilder.synchronized=true
 intersmash.bootable.jar.image=registry.access.redhat.com/ubi8/openjdk-17
 
 # DB
-intersmash.mysql.image=quay.io/centos7/mysql-80-centos7
-intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7
+intersmash.mysql.image=quay.io/centos7/mysql-80-centos7:centos7
+intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7:centos7
 
 intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17:latest
 intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17:latest

